NEXT_CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Release v0.248.0
 
 ### Notable Changes
+* Fixed a regression with pipeline library globs introduced in 0.247.0 ([#2723](https://github.com/databricks/cli/pull/2723)). The issue caused glob patterns to fail when using paths relative to a directory outside the bundle root.
```

Comment:
```suggestion
* Fixed a regression with pipeline library globs introduced in 0.247.0 ([#2723](https://github.com/databricks/cli/pull/2723)). The issue caused glob patterns to fail when using paths relative to a directory that is not the bundle root.
```

Otherwise it reads like the issue is when the directory is outside bundle root.